README.md

```diff
@@ -3,25 +3,26 @@
 Patchmanager is a tool that can be used to modify the Sailfish OS user experience.
 It is based on AUSMT (Auto-Update System Modification Technology), a set of scripts that enables system file patching.
 
-Patchmanager does not have application icon inside launcher. After installation Patchmanager can be found inside Settings.
+Patchmanager does not have application icon on the launcher.  After installation Patchmanager can be found inside Settings.
```

Comment:
nit: additional space

Reply:
Yes, well that happens if MRs are stalled for long: Information becomes outdated.
Thanks for noticing.
Fixed now.

Reply:
Didn't know all that:)